Makefile

```diff
@@ -238,6 +242,24 @@ $(CODER_ALL_BINARIES): go.mod go.sum \
 		cp "$@" "./site/out/bin/coder-$$os-$$arch$$dot_ext"
 	fi
 
+# This task builds Coder Desktop dylibs
+$(CODER_DYLIBS): go.mod go.sum $(GO_SRC_FILES)
```

Comment:
This needs to be phony since it doesn't always create the files. It would be better if this was a job that failed on non-linux, and then the tar/sha1 generation opportunistically included them if they existed

Reply:
> It would be better if this was a job that failed on non-linux
Will do
> then the tar/sha1 generation opportunistically included them if they existed
Yep, that's what it does currently